### PR TITLE
Fix manual components colour scheme

### PR DESCRIPTION
**Pull request by regiskuckaertz**
The refactored manual containers now support all of the varying colour schemes that are currently implemented in the legacy components.

cc @JonNorman @uplne

### CODE REVIEW

File: static/src/stylesheets/module/commercial/_adverts-books.scss [hunk 1]
```diff
@@ -69,6 +71,22 @@
     }
 }
 
+.adverts--tone-membership.adverts--legacy-single .adverts__row > .button {
```
Comment (uplne) [inline static/src/stylesheets/module/commercial/_adverts-books.scss]: what the...?

File: static/src/stylesheets/module/commercial/_adverts-shop.scss [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            @include button-colour(
+                darken($features-support-4, 5%),
+                null
+            );
+        }
+    }
+}
+
+.adverts--tone-shop.adverts--legacy-single .adverts__row > .button {
+    @include button-colour(
```
Comment (uplne) [inline static/src/stylesheets/module/commercial/_adverts-shop.scss]: There is a lot of repetition. Can't it be shared somehow? Possibly mixin?

Comment (regiskuckaertz) [inline static/src/stylesheets/module/commercial/_adverts-shop.scss]: I'll see what I can do

File: static/src/stylesheets/module/commercial/_adverts-books.scss (hunk 1 above)
Comment (regiskuckaertz) [inline static/src/stylesheets/module/commercial/_adverts-books.scss]: Yeah :-) I can do better

**Comment by regiskuckaertz** [comment]
What say you now @uplne ?

**Comment by uplne** [comment]
It is a little bit better :-) 👍

**Pull request merged**